### DISCUSSION TITLE
请求的时候怎么设置header头呢？

### DISCUSSION THREAD

**Discussion by linmao666** [w7corp/easywechat#2643] [Q&A]
请求的时候怎么设置header头呢？HTTP header缺少微信支付平台证书序列号(Wechatpay-Serial)

**Comment by TheNorthMemory** [comment]
https://easywechat.com/6.x/client.html

![image](https://user-images.githubusercontent.com/1631685/211255517-1f23ade9-2fca-498f-84a7-b31effa3e476.png)

```php
$client->post('v3/blah/blah', [
    'json' => ['a'=>'b'], 
    'headers' => ['Wechatpay-Serial' => 'ABEFDEABEFDEABEFDEABEFDEABEFDE'],
]);
```
